src/Nethermind/Nethermind.Blockchain/Tracer.cs

```diff
@@ -217,5 +216,17 @@ private ParityLikeTxTrace[] ParityTraceBlock(Block block, ParityTraceTypes trace
             _processor.Process(block, ProcessingOptions.ForceProcessing | ProcessingOptions.WithRollback | ProcessingOptions.ReadOnlyChain | ProcessingOptions.NoValidation, listener);
             return listener.BuildResult().ToArray();
         }
+        
+        private static Block GetBlock(Rlp blockRlp)
+        {
+            Block block = Rlp.Decode<Block>(blockRlp);
+            if (block.TotalDifficulty == null)
+            {
+                block.TotalDifficulty = 1;
```

Comment:
better leave empty?

Reply:
It was introduced like that in this commit: https://github.com/NethermindEth/nethermind/commit/b03faffe751ecb4509bf15ae20a858003898c274#diff-f46542ec72973c095e05ffdc4e7e1ca3 do you know why?

Reply:
Looked at it. Since the block is processed later it requires total diffoculty othereise blockchainprocessor would fail. My initial comment was here because I thoigh we return this block and not trace it.